synapse/server.py

```diff
@@ -221,6 +225,10 @@ def setup(self):
             conn.commit()
         logger.info("Finished setting up.")
 
+    def setup_master(self):
+        for i in self.REQUIRED_ON_MASTER_STARTUP:
+            getattr(self, "get_" + i)()
```

Comment:
Could  you doc some of this a little please.

Reply:
done